mandatory/1-fix-functions.js

```diff
@@ -10,7 +10,7 @@
 
 */
 
-function mood() {
+function mood(isHappy) {
```

Comment:
Hello Yousef. I think you should comment out line 14.

Reply:
Yes you are correct, thank you Shima